src/test/java/pl/coderslab/service/LoadingPlaceServiceTest.java

```diff
@@ -0,0 +1,133 @@
+package pl.coderslab.service;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import pl.coderslab.model.LoadingPlace;
+import pl.coderslab.repository.LoadingPlaceRepository;
+
+import javax.crypto.spec.OAEPParameterSpec;
+import javax.persistence.EntityNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class LoadingPlaceServiceTest {
+
+    private LoadingPlaceRepository loadingPlaceRepositoryMock = mock(LoadingPlaceRepository.class);
+    private LoadingPlaceService loadingPlaceService = new LoadingPlaceService(loadingPlaceRepositoryMock);
+
+    @Test
+    public void shouldAdd() {
+        //given
+        LoadingPlace loadingPlace = new LoadingPlace("firma", "43-433", "Strumień", "Strumień", "Polska", "firmatestowa");
+        loadingPlace.setId(2L);
+
+        //when
+        given(loadingPlaceRepositoryMock.findById(2L)).willReturn(Optional.of(loadingPlace));
+        loadingPlaceService.add(loadingPlace);
+
+        //then
+        LoadingPlace addedLoadingPlace = loadingPlaceService.showById(2L);
+        assertSame(addedLoadingPlace, loadingPlace);
+    }
+
+
+    @Test
+    public void shouldAddSecondWay() {
```

Comment:
co oznacza SecondWay widze, że tutaj sprawdzasz tą samą operacje czyli when jest to samo ale then będize inne bo sprawdzasz inne rzeczy i warto tak nazwać test